spacy/pipeline/entity_linker.py

```diff
@@ -41,6 +41,14 @@
 DEFAULT_NEL_MODEL = Config().from_str(default_model_config)["model"]
 
 
+"""
+[initialize.components.entity_linker.kb_loader]
+@misc = "spacy.CustomKB.v1"
+entity_vector_length = 342
+custom_field = 666
+"""
```

Comment:
I don't understand what this bit is doing here? It feels like a c/p from the test suite?